src/actions/notificationsActions.ts

```diff
@@ -1,13 +1,20 @@
-import { ActionTypes, INotificationsState } from "reducers/notificationsReducer";
+import { DismissNotification, ShowNotification } from "reducers/notificationsReducer";
 
-export const showAlert = (alert: string) => (dispatch: any) => {
-  const payload: INotificationsState = {
-    alert,
+export const showAlert = (message: string) => (dispatch: any) => {
+  const show: ShowNotification = {
+    type: "Notification/Show",
+    message,
```

Comment:
lets stick with redux best practice of having just a type and a payload object in the action, so the message and the timestamp would both be part of the payload